src/components/Help/HelpPanel.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Typography,
+} from '@material-ui/core';
+import { PAGES_TAB_NAMES } from 'utils/constants';
+import { UserMountsStepper } from 'components/Help/Steppers/UserMountsStepper';
+
+export const HelpPanel = () => {
+        const [expanded, setExpanded] = React.useState(false);
+      
+        const handleChange = (panel) => (event, isExpanded) => {
+          setExpanded(isExpanded ? panel : false);
+        };
+
+    return (
+      <div>
+        <h1 align='center'>{PAGES_TAB_NAMES.HELP_NAV}</h1>

```

Comment:
Can you use [Typography](https://material-ui.com/components/typography/) from MUI instead? This is very out of place because it has a different font.

Reply:
corrected